active/Always-on-top/README.md

```diff
@@ -0,0 +1,111 @@
+
+![Figure 1 - Default size](./imgs/figure_1.png)
+> Figure 1 - Default size
+
+![Figure 2 - Default size (left scroll arrow shown)](./imgs/figure_2.png)
+> Figure 2 - Default size (left scroll arrow shown)
+
+![Figure 3 - Hover over the chevron arrow](./imgs/figure_3.png)
+> Figure 3 - Hover over the chevron arrow
+
+![Figure 4 - Default size (right scroll arrow shown)](./imgs/figure_4.png)
+> Figure 4 - Default size (right scroll arrow shown)
+
+### Keyboard shortcuts
+All keyboard shortcuts for the mode other than standard are disabled in the always-on-top mode. The table below lists the standard keyboard shortcuts that are supported in the always-on-top mode.
+
+> Note: The keyboard shortcuts only supported in the regular and max-size layouts are called out in the table.
+
+| Shortcut | Action |
+|----------|-------------|
+| % | Select % (only supported in regular and max size layouts) |
+| @ | Calculate the square root (only supported in regular and max size layouts) |
+| q | Select x2 (only supported in regular and max size layouts) |
+| r | Select 1/X (only supported in regular and max size layouts) |
+| Esc | Clear all input (select C) |
+| Delete | Clear current input (select CE) |
+| Backspace | Delete last character |
+| / | Divide |
+| * | Multiply |
+| - | Subtract |
+| + | Add |
+| F9 | Select ± |
+| . | Decimal point |
+| Enter | Equal |
+
+### Window sizes & management
+Window resizing is enabled in the always-on-top mode. The default window size for the always-on-top mode is 320 x 394 epx. The minimum size of the window is 150 x 150 epx. The maximum size is 500 x 500 epx or half of the screen height depending on which is smaller.
+
+When the user shrinks the window down from the default size, the %, √, x ², 1/x keys are removed.
+
+![Figure 5 - Min-size (150x150 epx)](./imgs/figure_5.png)
+> Figure 5 - Min-size (150x150 epx)
+
+![Figure 6 - Min-size (150x150 epx)](./imgs/figure_6.png)
+> Figure 6 - Max-size (500 x 500 epx)
+
+If the user resizes the always-on-top window, the user configured size is saved and used as the default size the next time user enters the always-on-top mode (both in the same session and in new sessions).
+
+Window snapping is disabled in the always-on-top mode.
+
+### Enter Always-on-top mode
+Always-on-top button only shows in the standard mode. Clicking the always-on-top button ![Enter Always-on-Top icon](./imgs/enter_AoT_icon.png)  in the UI enables the always-on-top mode and places the always-on-top mode window at the upper right corner (position managed by the Compact Overlay mode API) of the screen. If another app is already in Compact overlay mode, Calculator always-on-top window is stacked on top of the other window and offset to the lower left direction to ensure that the first window is still partially visible (Compact Overlay mode API manages the positioning of the windows).
```

Comment:
Why limit the entry point to only Standard Mode? IMO, it should be available for Scientific mode as well to prevent users from having to discover they have to switch modes to activate.

Reply:
Since the button layouts are limited due to smaller size of compact overlay, we want to start with just Standard to prevent confusion over not seeing full scientific/programmer functionality if users were to enter from other modes.